pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["setuptools>=69", "wheel"]
```

Comment:
You removed wheel with a newer setuptools (70? 71?) in another PR. Can this be done here as well?